pkg/bundle/publish/publish.go

```diff
@@ -374,13 +376,14 @@ func (o *Options) Run(ctx context.Context, cmd *cobra.Command, f kcmdutil.Factor
 		catOpts.DryRun = o.DryRun
 		catOpts.MaxPathComponents = 2
 		catOpts.SecurityOptions.Insecure = o.SkipTLS
-		//catOpts.FilterOptions = imagemanifest.FilterOptions{FilterByOS: ".*"}
+		catOpts.FilterOptions = imagemanifest.FilterOptions{FilterByOS: o.FilterByOS}
```

Comment:
Another (non-blocking) thought: this could be done in the `create` step right?

Reply:
I believe so. That’s why I put it in root options. Can be done in a follow on PR :).